Form-Controls/index.html

```diff
@@ -21,7 +59,7 @@ <h1>Product Pick</h1>
     </main>
     <footer>
       <!-- change to your name-->
-      <h2>By HOMEWORK SOLUTION</h2>
+      <h2>BY Priscilla Emebo</h2>
```

Comment:
I don't think that heading level 2 is appropriate semantic tag here. Please think of a better tag for this case.